code/script.js

```diff
@@ -0,0 +1,135 @@
+// Token
+const options = {
+    method: 'GET',
+    headers: {
+        Authorization: `TOKEN ${API_TOKEN}`
+    }
+}
+
+// DOM selectors
+const projectInfo = document.getElementById('projectInfo')
+const userInfo = document.getElementById('userInfo')
+const aboutUser = document.getElementById('aboutUser')
+const profilePicture = document.getElementById('profilePicture')
+const mainContent = document.getElementById('mainContent')
+
+
+// Github API
+const username = 'emmajosefina'
+const API_URL = `https://api.github.com/users/${username}/repos`
+const API_USER = `https://api.github.com/users/${username}`
+
+
+
+ //------------------ FIRST FETCH - USER PROFILE -----------------------//
+const getProfile = () => {
+    fetch(API_USER, options)
+      .then((res) => res.json())
+      .then((data) => {
+        mainContent.innerHTML +=
+        `
+        <section class="profile-box">
+            <img src="${data.avatar_url}" id="profilePicture" class="profile-picture" />
+            <p class="full-name">${data.name}</p>
+            <p class="username">${data.login}</p>
+        </section>
+        <div class="status-box">
+        ${data.bio} 👩‍💻
+        </div>
+          `;
+      });
+  };
+  getProfile();
+
+
+   //------------------ SECOND FETCH - ALL REPOS -----------------------//
+const findingAllRepos = (repos) => {
+    fetch(API_URL, options)
+       .then((res) => res.json())
+       .then((data) => {
+
+
+
+    // Fetches only repositories from Technigo //
+    const forkedRepos = data.filter((repo) => repo.fork && repo.name.startsWith('project-'))
+
+
+    forkedRepos.forEach((repo) =>
+    projectInfo.innerHTML +=
+
+    `
+    <div class="projectContainer">
+        <p class="smallerContainer">
+            <span class="header-project">
+            ${repo.name.replace('project-', '').replace('-', ' ')}
```

Comment:
Really nice removing "project" and the hyphens, I never got around to that myself.